bevy_lint/src/lint.rs

```diff
@@ -163,25 +156,9 @@ macro_rules! declare_bevy_lint_pass {
     (
         $(#[$attr:meta])*
         $vis:vis $name:ident => [$($lint:expr),* $(,)?],
-
-        $(
-            @default = {
-                $($default_field:ident: $default_ty:ty = $default_value:expr),* $(,)?
-            },
-        )?
```

Comment:
We no longer need `@default`, as its primary purpose was to intern symbols we used. We now have the `sym` module for that!